Python/pathconfig.c

```diff
@@ -741,7 +741,9 @@ _Py_FindEnvConfigValue(FILE *env_file, const wchar_t *key,
     char buffer[MAXPATHLEN * 2 + 1];  /* allow extra for key, '=', etc. */
     buffer[Py_ARRAY_LENGTH(buffer)-1] = '\0';
 
```

Comment:
Please update the function comment: explain that it returns 1 and copy it into value if found, or return 0 on error or if not found. Something like that.

Reply:
Note: I'm not sure that fseek(0) is done: we just opened the file in read mode, not in append mode, the file cursor should already be 0.